src/components/ExpiryNotification/TruncatedAddress.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+export default function TruncatedAddress({ address = '' }) {
+  // const [showFullAddress, setShowFullAddress] = useState(false)
```

Comment:
Is this commented out for some sort of debugging purpose?

Reply:
Removed - Originally I had provided the ability to expand a truncated address by clicking on the ellipsis, but this caused layout issues within the modal. I left this in place in-case we wanted to revisit that capability later.